### PR TITLE
Update README.md

### LINKED ISSUE #29: Add your name to the list of contributors
Add your name to the list of contributors to test your repository access, and that you can run the project!

DoD:

- You have added your name and GitHub username to the list of contributors
- You have run the project
- You have checked out the code in the project and have a basic understanding of it
- You have asked any questions to @ChristianBingman

### PR DESCRIPTION
**Pull request by Mohpann**
Added name to contributors list.
resolves #29

### CODE REVIEW

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Make sure you tag your issue with `resolves #issue num`

File: README.md [hunk 1]
```diff
@@ -72,3 +72,4 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/t
 ## Contributors
 
 Christian Bingman - [@ChristianBingman](https://github.com/ChristianBingman)
+Connor Hunter - [@Mohpann](https://github.com/Mohpann)
```
Comment (Mohpann) [inline README.md:75]:
```suggestion
Connor Hunter - [@Mohpann](https://github.com/Mohpann) 
resolves #29 
```

**Comment by ChristianBingman** [comment]
I am going to close this since you are supposed to pick up an existing issue, not create a new one.

**Comment by ChristianBingman** [comment]
The resolves needs to go in the title or description of the PR

File: README.md (hunk 1 above)
Comment (ChristianBingman) [reply README.md:75]: This should be in the commit message, not on the README

**Review by ChristianBingman — APPROVED** [review]
Great! Thank you!

**Pull request merged**